Error: reference to 'byte' is ambiguous

When compiling qzxing for arm android using Qt 5.12, NDK r16b, Clang and c++17 flag needed for my project, compiler throws error:
```
qzxing/src/zxing/zxing/common/DecoderResult.cpp:40:10: error: reference to 'byte' is ambiguous
ArrayRef<byte> DecoderResult::getRawBytes() {
         ^
/usr/local/share/ndk/android-ndk-r16b/sources/cxx-stl/llvm-libc++/include/cstddef:65:12: note: candidate found by name lookup is 'std::byte'
enum class byte : unsigned char {};
           ^
qzxing/src/zxing/zxing/common/Types.h:11:23: note: candidate found by name lookup is 'zxing::byte'
typedef unsigned char byte;
                      ^
```

--- a/src/zxing/zxing/common/DecoderResult.cpp
+++ b/src/zxing/zxing/common/DecoderResult.cpp
@@ -37,7 +37,7 @@ DecoderResult::DecoderResult(ArrayRef<byte> rawBytes,
                              Ref<String> text)
   : rawBytes_(rawBytes), text_(text),charSet_("") {}
 
-ArrayRef<byte> DecoderResult::getRawBytes() {
+ArrayRef<zxing::byte> DecoderResult::getRawBytes() {
   return rawBytes_;
 }
 

